unitOneAssessmentPractice.js

```diff
@@ -3,27 +3,53 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
+const double = (num) => {
+  return num * 2
+}
 
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
+const containsSeven = (num) => {
+  return num.toString().includes('7');
+}
+
 // Uncomment out the next line to test your solution
-// runQ2Tests()
+runQ2Tests()
 
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
 
+const capitalizeTheAs = (string) => {
+  let newStr = "";
+  for(let i = 0; i < string.length; i++){
+    if(string[i] === "a") {
+      newStr += string[i].toUpperCase()
+    } else {
+      newStr += string[i]
+    }
+  }
+  return newStr
+}
+  
+
 // Uncomment out the next line to test your solution
-// runQ3Tests()
+runQ3Tests()
 
 // Question Four:
 // Write a function called largest that returns the largest value in an array
 
+// const largest = (arr) => {
+//   for(let i = 0; i < Infinity; i++) {
+//     if(arr[i] > )
```

Comment:
You're close, try creating a largest variable that keeps track of the largest variable you've seen so far